ARC-simple/src/main/kotlin/parts/wisdom/arcsimple/ArkSimple.kt

```diff
@@ -0,0 +1,33 @@
+package parts.wisdom.arcsimple
```

Comment:
I'm seeing the functionality of this file as part of the base ARC raider project. I was only thinking we might want a subproject for classes specific to our simplified ARC world. But at the moment I'm thinking even that is wrong-headed, since the tiny problem space of the simplified ARC world is a legitimate subspace of the full ARC world.

Reply:
I'm thinking we can solve for the simple example first, and then we when want to generalize more we can refactor and pull the general parts back into the main module. ie this will eventually just contain classes specific to the simplified case, and if that ends with no simple specific classes eventually this module will go away.

Reply:
I'm fine with that.

Reply:
In the light of morning, I'm feeling more uncomfortable rather than less uncomfortable with cloning the Ark.kt classes into the ARC-simple subproject. Seems like we will end up ignoring the original Ark.kt while we further develop ArkSimple.kt, then copy the latter into the former and lose commit history. And what happens if we don't have  a sharp break between working on the simple ARC world and starting to try things on the official ARC problems? During that overlap period, do we start developing against official ARC problems in ARC-simple? Or do we repeatedly copy-paste ArkSimple.kt back to Ark.kt?